### ISSUE TITLE
'search_cord19.py' only outputs number of datapacks, without response to query

### ISSUE THREAD

**Issue by samunaai** [petuum/composing_information_system#22]
**Context**
Hello There!👋🏿👋🏿
I am facing an issue in running the code for the QA engine in the 3-part Forte [tutorial](https://medium.com/casl-project/building-a-question-answering-system-part-1-query-understanding-in-18-lines-916110f9f2b2), also described in the [QA.md](https://github.com/petuum/composing_information_system/blob/main/QA.md).

**Issue**
When I run `python examples/pipeline/inference/search_cord19.py`, it appears the python file [`search_cord19.py`](https://github.com/petuum/composing_information_system/blob/main/examples/pipeline/inference/search_cord19.py)  is written (in the `__main__` loop) to only output the _number_ of datapacks (= 11 in the screenshot below), instead of the _response_ to the query.

This seems different from other Forte based examples such as the Forte Chatbot [[link](https://github.com/asyml/forte/blob/master/examples/chatbot/chatbot_example.py)], whereby the output datapacks obtained from `query_pipeline.process_dataset()`, are further processed with more lines of code, before the chatbot output is shown.

In other words, it appears that some code in the last part of QA python file may have been ommitted?  This was also the case in prior version of the `search_cord19.py` found in the [commit history](https://github.com/petuum/composing_information_system/commits/main/examples/pipeline/inference/search_cord19.py), as well as the 3-part blog post. Perhaps there may be some lines of code I can add to the python file, to allow for the answer to be displayed in response to a query?

Warm regards,

**Screenshots**
![image](https://user-images.githubusercontent.com/22077758/159328607-0a7a9cdb-0c8e-47b8-bc7d-30401635504b.png)

![image](https://user-images.githubusercontent.com/22077758/159360595-d90c098c-87ba-48a4-888d-42bdede51c92.png)

**Comment by samunaai** [comment]
Hello @feipenghe  👋🏿👋🏿
I am now getting the new error below:
It appears to be caused by the package name change of `ftx.medical.clinical_ontology` to `ftx.onto.clinical` that @hunterhector explained [here](https://github.com/asyml/forte-wrappers/issues/98)
Actually when I go to the  file `.../python3.6/site-packages/fortex/spacy/spacy_processors.py` I notice that it says 
`"medical_onto_type": "ftx.medical.clinical_ontology."` in the `default_configs` method on line 354 .. 
However, changing this did not immediately solve the problem .. Would you have any idea how we can correct this package name change issue?
Warm regards,
![image](https://user-images.githubusercontent.com/22077758/161801232-e55233e7-9fed-467c-954a-b145978661b4.png)
